src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -53,12 +64,14 @@ public SonarQubeIssuesReport extractIssueReport(ProjectIssues projectIssues, Inp
   /**
    * Post SQ analysis overview on Stash
    */
-  public void postAnalysisOverview(String project, String repository, String pullRequestId, String sonarQubeURL, int issueThreshold, SonarQubeIssuesReport issueReport, StashClient stashClient){
+  public void postAnalysisOverview(String project, String repository, String pullRequestId, String sonarQubeURL, String stashURL,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)